internal/util/sizeof_test.go

```diff
@@ -294,10 +308,11 @@ func (t *SizeofTest) TestNestedSizeOfGcsMinObject() {
 		Generation:      generation,
 		MetaGeneration:  metaGeneration,
 		Updated:         updated,
+		CRC32C:          &crc32,
```

Comment:
Didn't notice it earlier. It's probably outside the scope of this review. But just curious. What does the trailing `C` in CRC32**C** stand for?

Reply:
Also, the local variable on the RHS should also get a trailing `c` then.

Reply:
crc32c stands for Castagnoli CRC32, a different polynomial used for computing the checksum
I believe it should be fine to not have c in crc32c at every place since gcs only uses crc32c

Reply:
> I believe it should be fine to not have c in crc32c at every place since gcs only uses crc32c

sounds good.